src/components/quote.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Quote = () => (
+  <div>
+    <p className="quote">Mathematics is not something about numbers, equations, computaions,or algorithm: </p>
```

Comment:
- [ ] Optional suggestion

   - Kindly, keep one `<p>` tag, with all the text, Imagine once you have more than 100 lines with the `<p>` tag.

Reply:
Thank you, I will fix that